fastqc

```diff
@@ -346,13 +346,8 @@ if (@files or $version or $help) {
 
 #warn "Running 'exec $java_bin,@java_args, \"uk.ac.babraham.FastQC.FastQCApplication\", @files, CLASSPATH is $ENV{CLASSPATH}";
 
-
-if ($java_bin ne 'java') {
-	system $java_bin,@java_args, "uk.ac.babraham.FastQC.FastQCApplication", @files;
-}
-else {
-	exec $java_bin,@java_args, "uk.ac.babraham.FastQC.FastQCApplication", @files;
```

Comment:
`exec` is designed to be very blind to the success or failure of the underlying process. Please review this change as I haven't seen `system` and `exec` mixed like this before.